plugin/src/main/java/io/jenkins/plugins/casc/Attribute.java

```diff
@@ -186,11 +185,9 @@ public CNode describe(Owner instance, ConfigurationContext context) throws Confi
                 return seq;
```

Comment:
Might actually be better to `catch` things at line 183 above, in case there are some elements of a collection which can be described and others which cannot. I am not quite sure what that would look like in YAML though.